### PR TITLE
tg3 parameters for  driver_parameter.py

### PR DESCRIPTION
**Pull request by vaishnavibhat**
yaml file for tg3 driver

Signed-off-by: Vaishnavi Bhat <vaishnavi@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by narasimhan-v** [comment]
@vaishnavibhat please upload job.log

**Comment by vaishnavibhat** [comment]
# avocado run driver_parameter.py -m driver_parameter_tg3.yaml 
/usr/local/lib/python3.6/site-packages/avocado_framework-75.1-py3.6.egg/avocado/plugins/run.py:214: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast, keep-tmp and ignore-missing-references.
  FutureWarning)
JOB ID     : bc3f91ca4ba85db49e263b6ebac422b6106495d0
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2020-02-18T19.29-bc3f91c/job.log
 (1/2) driver_parameter.py:Moduleparameter.test;0-d3ed: PASS (73.32 s)
 (2/2) driver_parameter.py:Moduleparameter.test;1-2ccc: PASS (73.61 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 147.42 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2020-02-18T19.29-bc3f91c/results.html
[tg3_parm.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4219640/tg3_parm.log)

File: io/driver/driver_parameter.py.data/driver_parameter_tg3.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module: tg3
+interface: eth1
+host_ip:
+netmask:
+peer_ip:
+sysfs_check_required: 0
+module_param_name: "tg3_debug"
+value: !mux
+    0:
```
Comment (narasimhan-v) [inline io/driver/driver_parameter.py.data/driver_parameter_tg3.yaml]: can this be renamed to "tg3_debug-0"

File: io/driver/driver_parameter.py.data/driver_parameter_tg3.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+       module_param_value: "0"
+    1:
```
Comment (narasimhan-v) [inline io/driver/driver_parameter.py.data/driver_parameter_tg3.yaml]: can this be renamed to "tg3_debug-1"

**Pull request merged**